torch/_torch_docs.py

```diff
@@ -1352,37 +1352,50 @@ def merge_dicts(*dicts):
 Args:
     input (Tensor): the tensor to split
     indices_or_sections (int or (list(int))):
```

Comment:
`indices_or_sections (int or (list(int))):` -> `indices_or_sections (Tensor, int or list or tuple of ints):`